server.go

```diff
@@ -187,20 +187,22 @@ func listen(conn net.PacketConn, tlsConf *tls.Config, config *Config, acceptEarl
 		}
 	}
 
-	sessionHandler, err := getMultiplexer().AddConn(conn, config.ConnectionIDLength, config.StatelessResetKey, config.Tracer)
+	conn, err := wrapConn(pconn)
 	if err != nil {
 		return nil, err
 	}
-	tokenGenerator, err := handshake.NewTokenGenerator(rand.Reader)
+
+	sessionHandler, err := getMultiplexer().AddConn(conn, config.ConnectionIDLength, config.StatelessResetKey, config.Tracer)
```

Comment:
…since `AddConn` calls `wrapConn`, which will fall through because `conn` is already wrapped.

Reply:
... which never happens, because `listen` is only called from `Listen{Addr}{Early}`. Or am I missing something?

Reply:
1. `listen` calls `(*connMultiplexer).AddConn`, which calls `newPacketHandlerManager` (`newPacketHandlerMap`), which calls `wrapConn`.
2. `listen` also calls `wrapConn` itself on the same `net.UDPConn`.

That said, this doesn’t matter because `oobConn` isn’t thread-safe. That’s why I closed this PR.